casper/src/main/scala/coop/rchain/casper/ValidatorIdentity.scala

```diff
@@ -59,11 +56,6 @@ object ValidatorIdentity {
     )
   }
 
-  def getEnvVariablePassword[F[_]: Sync: EnvVars]: F[String] =
-    EnvVars[F].get(RNodeValidatorPasswordEnvVar) >>= (
-      _.liftTo(new Exception(s"Environment variable $RNodeValidatorPasswordEnvVar is unspecified"))
-    )
-
```

Comment:
That's a nice code cleanup, but does it apply to this pull request? I think it's better to mark somewhere all the code fragments that need to be cleaned up, and then clean up the code with a separate pull request.

Reply:
Extracted to separate PR:
- #3726.